### DISCUSSION TITLE
Repository specific configuration in global config.js

### DISCUSSION THREAD

**Discussion by loadi-dev** [renovatebot/renovate#14412] [Request Help] [answered]
Hey there,

I'm using Renovate Bot (latest) on a self-hosted GitLab instance. So far I managed to put all the configuration needed into the `config.js` file in the renovate repository holding my CI configuration. I 'm pretty happy with this setup, as I don't need to add any "external" files to my applications and I have full control over renovate looking at just one place.

I now would like to introduce a setting (`bumpVersion`) for just one repository in my `repositories` array without affecting the others. Is this somehow possible, without adding a `renovate.json` to my applications repository or setting up a separate CI job (keeping it in `config.js`)?

Thanks in advance,
loadi

**Comment by viceice** [comment, marked as answer]
the `repositories` array can be object:

```js
{
  repositories: [ { repository: 'g/r', bumpVersion: true } ],
}
```

@HonkingGoose @rarkins We should add this to the docs here:

https://github.com/renovatebot/renovate/blob/main/docs/usage/self-hosted-configuration.md#repositories

**Reply by viceice** [reply]
here's an example https://github.com/renovatebot/renovate/blob/main/docs/usage/self-hosted-configuration.md#secrets
https://docs.renovatebot.com/self-hosted-configuration/#secrets

**Reply by loadi-dev** [reply]
Ah, nice. Thanks a lot!
